react/features/toolbox/functions.native.js

```diff
@@ -5,6 +5,52 @@ import { TOOLBOX_ALWAYS_VISIBLE, getFeatureFlag, TOOLBOX_ENABLED } from '../base
 import { toState } from '../base/redux';
 import { isLocalVideoTrackDesktop } from '../base/tracks';
 
+const WIDTH = {
+    FIT_8_ICONS: 500,
+    FIT_7_ICONS: 440,
+    FIT_6_ICONS: 380,
+    FIT_5_ICONS: 320
+};
+
+/**
+ * Returns a set of the buttons that are shown in the toolbar
+ * but removed from the overflow menu, based on the width of the screen.
+ *
+ * @param {number} width - The width of the screen.
+ * @returns {Set}
+ */
+export function getMovableButtons(width: number): Set<string> {
+    let buttons = [];
+
+    switch (true) {
+    case width >= WIDTH.FIT_8_ICONS: {
+        buttons = [ 'chat', 'invite', 'raisehand', 'tileview' ];
+        break;
+    }
+
+    case width >= WIDTH.FIT_7_ICONS: {
+        buttons = [ 'chat', 'invite', 'raisehand' ];
+        break;
+    }
+
+    case width >= WIDTH.FIT_6_ICONS: {
+        buttons = [ 'chat', 'invite' ];
```

Comment:
Can you switch invite with raise hand here? I think it's more useful to have that one if  there is room for it.

Reply:
Done.